src/components/Calculator.js

```diff
@@ -15,6 +15,7 @@ const Calculator = () => {
 
   return (
     <div className="h-screen flex items-center justify-center bg-blue-600">
+      <h3 className="ml-2 font-bold text-white dark:text-gray-400 bg-green-500 p-6 rounded-lg shadow-lg  margin-left: 0.125rem; absolute top-40 left-3.5 text-3xl">Let s do some Math !</h3>
       <div className="bg-white p-6 rounded-lg shadow-lg">
         <div className="grid grid-cols-4 gap-0.5">
           <div
```

Comment:
- [ ] @Bleu-Yves-Sopoude Great work on your calculator page, however, the layout doesn't match the design given kindly check the comparison below, the position of the calculator:

Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/Bleu-Yves-Sopoude/maths-magicians/assets/61408860/45d0b639-904f-4220-834d-531bb058abb7)  |  ![image](https://github.com/Bleu-Yves-Sopoude/maths-magicians/assets/61408860/1b0d5af8-cf23-4f10-b4cd-d7a940f41866)